sql/accounts/accounts_test.go

```diff
@@ -48,6 +48,63 @@ func TestHas(t *testing.T) {
 	require.True(t, has)
 }
 
+func TestLatest(t *testing.T) {
+	t.Run("doesn't exist", func(t *testing.T) {
+		db := statesql.InMemoryTest(t)
+		account, err := Latest(db, types.RandomAddress(t))
+		require.ErrorIs(t, err, sql.ErrNotFound)
+		require.Equal(t, types.Account{}, account)
```

Comment:
NIT: whenever you compare agains the empty object you can do

```suggestion
		require.Empty(t, account)
```